public/style.css

```diff
@@ -228,7 +228,7 @@ body {
   margin-left: -1px;
   padding-top: 13px;
   border: 0;
-  background-color: #00b0ff;
+  background-color: #007bff;
```

Comment:
Remove this color from here and add `bg-primary` class to respective places in code.